onnx_tf/handlers/backend/resize.py

```diff
@@ -0,0 +1,60 @@
+import copy
+
+import numpy as np
+import tensorflow as tf
+
+from onnx_tf.common import exception
+from onnx_tf.handlers.backend_handler import BackendHandler
+from onnx_tf.handlers.handler import onnx_op
+
+
+@onnx_op("Resize")
+class Resize(BackendHandler):
+
+  @classmethod
+  def args_check(cls, node, **kwargs):
+    x = kwargs["tensor_dict"][node.inputs[0]]
+    x_shape = x.get_shape().as_list()
+    if len(x_shape) != 4:
+      exception.OP_UNSUPPORTED_EXCEPT("Resize required 4D input", "Tensorflow")
+
+  @classmethod
+  def version_10(cls, node, **kwargs):
+    x = kwargs["tensor_dict"][node.inputs[0]]
+    x_shape = x.get_shape().as_list()
+    scales = kwargs["tensor_dict"][node.inputs[1]]
+    attrs = copy.deepcopy(node.attrs)
```

Comment:
Looks like no need to make a copy. Should be able to use node.attrs directly.

Reply:
@chinhuang007 Done, please review again.